src/AppBundle/EventListener/CheckSQLiteEventSubscriber.php

```diff
@@ -57,4 +61,16 @@ public function handleDatabaseExceptions(ConsoleExceptionEvent $event)
             }
         }
     }
+
+    /**
+     * This method is triggered when kernel exception occurs. And checks if sqlite extension is enabled.
+     *
+     * @param GetResponseForExceptionEvent $event
+     */
+    public function onKernelException(GetResponseForExceptionEvent $event)
+    {
+        if (!extension_loaded('sqlite3')) {
```

Comment:
Yes, i agree with that. But the problem is that from $event->getException() i get _Twig_Error_Runtime_ and don't know why. Maybe is documented somewhere?